### PR TITLE
feat: fast withdraw sdk

### PR DESCRIPTION
**Pull request by zhangyouxin**
- feat: fast withdraw sdk
- feat: use codec for v0 data structure
- refactor: remove unused import
- feat: install @ckb-lumos/codec
- fix: remove duplicate import
- fix: fix @ckb-lumos/bi dependency
- chore: change @ckb-lumos deps version to 0.17-rc10

### CODE REVIEW

File: src/light-godwoken/LightGodwokenV0.ts [hunk 1]
```diff
@@ -227,44 +224,117 @@ export default class DefaultLightGodwokenV0 extends DefaultLightGodwoken impleme
...
-      rawWithdrawalRequest,
+      new toolkit.Reader(RawWithdrwalCodec.pack(rawWithdrawalRequest)).serializeJson(),
       layer2Config.ROLLUP_CONFIG.rollup_type_hash,
     );
-    debug("message:", message);
+    console.log("message:", message);
     const signatureMetamaskPersonalSign: HexString = await this.signMessageMetamaskPersonalSign(message);
-    debug("signatureMetamaskPersonalSign:", signatureMetamaskPersonalSign);
-    const withdrawalRequest: WithdrawalRequest = {
+    console.log("signatureMetamaskPersonalSign:", signatureMetamaskPersonalSign);
```
Comment (sighWang) [inline src/light-godwoken/LightGodwokenV0.ts:302]: change console to debug

File: src/light-godwoken/schemas/codec.ts [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    return new Uint8Array(data.buffer);
+  },
+  unpack: (buf) => {
+    const data = new DataView(buf).getUint8(0);
+    if (data === 0) {
+      return "data";
+    } else if (data === 1) {
+      return "type";
+    } else if (data === 2) {
+      return "data1";
```
Comment (sighWang) [inline src/light-godwoken/schemas/codec.ts:29]: why named `data1 `here?

File: src/light-godwoken/constants/lightGodwokenConfig.ts [hunk 3]
```diff
@@ -40,42 +40,78 @@ export type LightGodwokenConfig = {
...
+        code_hash: "0x5c365147bb6c40e817a2a53e0dec3661f7390cc77f0c02db138303177b12e9fb",
+        hash_type: "type",
+        args: "0x213743d13048e9f36728c547ab736023a7426e15a3d7d1c82f43ec3b5f266df2",
+      },
     },
+    GW_POLYJUICE_RPC_URL: "https://godwoken-testnet-web3-rpc.ckbapp.dev",
+  },
+};
+
+export const predefined_v1_1: Record<GodwokenVersion, LightGodwokenConfig> = {
```
[the remark this answers is not in the document]
Comment (zhangyouxin) [reply src/light-godwoken/constants/lightGodwokenConfig.ts:104]: meant to be v1.1, but `.` is not allowed, any suggestions?

File: src/light-godwoken/schemas/codec.ts (hunk 2 above)
Comment (zhangyouxin) [reply src/light-godwoken/schemas/codec.ts:29]: reference: https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0032-ckb-vm-version-selection/0032-ckb-vm-version-selection.md#specification

File: src/light-godwoken/constants/lightGodwokenConfig.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (sighWang) [reply src/light-godwoken/constants/lightGodwokenConfig.ts:104]: maybe add a comment here

**Review by sighWang — APPROVED** [review]

**Pull request merged**